### PR TITLE
Nitpicks

### PR DESCRIPTION
**Pull request by sdzivanovich**
Relatively minor changes that you may or may not find useful

### CODE REVIEW

File: text/10_what_is_software_testing.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Let's start with a few things that it is not.
 2. It's not randomly pressing buttons, hoping that something will break.
 3. It's not hoping that something will break, period.
 4. It's not something you do after all the programming is complete.
-5. It's really, REALLY not something you don't think about until users start complaining.
+5. It's really, REALLY not something you postpone until users start complaining.
```
Comment (sdzivanovich) [inline text/10_what_is_software_testing.md]: Using "postpone" removes the double negative

File: text/10_what_is_software_testing.md [hunk 2]
```diff
@@ -42,9 +42,9 @@ Keep in mind that I am someone who teaches a class on software testing.  Even I
 
 ## Verification and Validation
 
-Software testing also involves ensuring that the right software was created.  Imagine the following conversation between a tester and a customer:
+Software testing also involves ensuring that the right software was created.  Imagine the following conversation between a project manager and a customer:
```
Comment (sdzivanovich) [inline text/10_what_is_software_testing.md]: Maybe this is just a difference in experience, but would a tester actually have a conversation like this with a customer?

File: text/10_what_is_software_testing.md [hunk 3]
```diff
@@ -67,11 +67,11 @@ For example, consider a company creating a brand-new Tic-Tac-Toe implementation.
...
 4. Players shall then take turns placing X's and O's (by the first and second player, respectively) in open squares, until there are no open squares and no row, column or diagonal is filled in with the same marker (in which case the game is a draw) or an entire row, column, or diagonal is filled in with the same marker, in which case the owner of that marker (X for first player, O for second player) shall be the winner and the other player shall be the loser.
 
-This sums up the game of Tic-Tac-Toe pretty nicely.  Now let's say that after beta testing, a user says that the game is unfair, because it forces one player to use X's and that mark is ugly.  The user suggests changing the X's into W's, because W is a much more beautiful letter.  Is this a defect or enhancement?
+This sums up the game of Tic-Tac-Toe pretty nicely. Now let's consider an instance where the first player, who should only be able to mark squares with X's, can mark a square with an O.  This is a defect, because it violates Requirement 2.  Even if the game is perfectly playable (let's say that the second player's marks then become X's), it's still a defect because it violates a requirement.
```
Comment (sdzivanovich) [inline text/10_what_is_software_testing.md]: Here I just reordered the paragraph so the example of something that *is* a defect comes before an example of something that isn't.

File: text/15_testing_basics.md [hunk 4]
```diff
@@ -153,9 +153,9 @@ Consider a desktop email application.  Tasked with testing this, a black box tes
 
 Aspects of the system, such as what algorithm was used or what kind of memory allocation scheme is used, can be inferred by the black-box tester, but their concern is focused on the results of the system running.  For example, a black-box tester might notice that the system gets extremely slow when sorting a user's inbox when the user has thousands of messages.  The reason for this is a poor algorithm choice, but the tester is concerned with reporting the result that it is slow, not looking up or analyzing the algorithm.
 
-One could consider __white box testing__ to be the opposite of black box testing.  In white box testing, the tester has intimate knowledge of the codebase and tests code itself.  White box tests can test individual functions of the code, often looking at much more granular aspects of the system than black box tests.
```
Comment (sdzivanovich) [inline text/15_testing_basics.md]: "One could consider" isn't very definitive wording

File: text/15_testing_basics.md [hunk 5]
```diff
@@ -153,9 +153,9 @@ Consider a desktop email application.  Tasked with testing this, a black box tes
...
+Continuing the example of a desktop email application, white box tests might check the actual `sort(EmailEntry[] emails)` function, sending in various values to see what the function returns or does.  White box testers would care about what happened specifically if a zero-length array or null reference were passed in, whereas a black box tester would only care about what happens if they attempt to sort an empty list of emails in the application itself.  White box tests access the code as code - checking that return values from functions are correct, ensuring that objects are instantiated properly, etc. - instead of looking at the system from a user's perspective.
```
Comment (sdzivanovich) [inline text/15_testing_basics.md]: This sentence was confusing for me. I think it's useful here to illustrate what the "edge case" actually looks like in a black box testing perspective

File: text/20_requirements.md [hunk 6]
```diff
@@ -83,7 +83,7 @@ Here is an example of a group of requirements that is not internally consistent:
...
 
-The requirements should be __unambiguous__.  That is, they should specify things as precisely as possible for working in the particular domain the software is for.  The acceptable level of ambiguity will vary dramatically depending on what kind of software you are developing.  For example, if you are writing software for a children's game, it may be sufficient to state that a particular square should be "red".  If you are describing the requirements for an interface to a nuclear reactor, the exact PANTONE® shade of red that a warning light should be may need to be specified.
+The requirements should be __unambiguous__.  That is, they should specify things as precisely as possible for working in the particular domain the software is for.  The acceptable level of ambiguity will vary dramatically depending on what kind of software you are developing.  For example, if you are writing software for a children's game, it may be sufficient to state that a particular square should be "red".  If you are describing the requirements for an interface to a nuclear reactor, the exact PANTONE® shade of red that a warning light may need to be specified.
```
Comment (sdzivanovich) [inline text/20_requirements.md]: Small typo fix

File: text/20_requirements.md [hunk 7]
```diff
@@ -108,7 +108,7 @@ When applied to physical phenomena, it is manifestly clear how silly it would be
 
 ## Functional versus Non-Functional
 
-Functional requirements state what a system should _do_; non-functional requirements state what a system should _be_.
+__Functional requirements__ state what a system should _do_; __non-functional requirements__ state what a system should _be_.
```
Comment (sdzivanovich) [inline text/20_requirements.md]: In the preceding chapters you bolded the terms that were important takeaways, so I did the same here for functional/non-functional requirements.

File: text/15_testing_basics.md (hunk 5 above)
Comment (laboon) [inline text/15_testing_basics.md]: Oh yes, much better with a concrete example.

File: text/10_what_is_software_testing.md (hunk 3 above)
Comment (laboon) [inline text/10_what_is_software_testing.md]: :+1:

File: text/10_what_is_software_testing.md (hunk 2 above)
Comment (laboon) [inline text/10_what_is_software_testing.md]: Occasionally, depending on the project, but you're right, a project manager is __much__ more likely to do so.

File: text/10_what_is_software_testing.md (hunk 1 above)
Comment (laboon) [inline text/10_what_is_software_testing.md]: THIS IS EXCELLENT

File: text/15_testing_basics.md (hunk 4 above)
Comment (laboon) [inline text/15_testing_basics.md]: !!! I know !!! I just couldn't think of a better way to phrase it at the time.  This is great!

File: text/20_requirements.md (hunk 7 above)
Comment (laboon) [inline text/20_requirements.md]: Bolded italics are used specifically for "the first time a new term is introduced".  I may want to mention that somewhere.

**Comment by laboon** [comment]
This is awesome, merging

**Pull request merged**